bazel/envoy_test.bzl

```diff
@@ -169,10 +169,12 @@ def envoy_cc_test(
         linkstatic = envoy_linkstatic(),
         malloc = tcmalloc_external_dep(repository),
         deps = envoy_stdlib_deps() + deps + [envoy_external_dep_path(dep) for dep in external_deps + ["googletest"]] + [
-            repository + "//test:test_pch",
             repository + "//test:main",
             repository + "//test/test_common:test_version_linkstamp",
-        ],
+        ] + select({
```

Comment:
Needs to be a list

Reply:
Sorry what do you mean? You need brackets around the select? Or inside the select?

Reply:
(I think we have brackets inside the select?)